code/noRAM/app/src/main/res/values/strings.xml

```diff
@@ -90,5 +90,6 @@
     <string name="have_fun_at_event_placeholder"><![CDATA[Have fun at <event>!]]></string>
     <string name="thanks_for_checking_in_header">Thanks For Checking In!</string>
     <string name="display_name">Display Name</string>
-    <string name="signedin_text">Signed In</string>
+    <string name="signedin_text">Signed Up</string>
+    <string name="nothing_to_show_text">Nothing to show...</string>
```

Comment:
Is this used for anything?

Reply:
ah oops, was going to try and add the nothing to show thing for listviews but then couldn't get it working and scrapped it for now. prob will delete